content/en/docs/v3.5/tutorials/how-to-transactional-write.md

```diff
@@ -17,22 +28,35 @@ weight: 500
 etcdctl txn --help
 ```
 
-Transactions in etcd allow you to execute multiple operations atomically, ensuring that either all operations are applied or none are. This is crucial for maintaining data consistency when performing related updates.
+Transactions in etcd allow you to execute multiple operations atomically, ensuring that either all operations are applied or none are. This is crucial for maintaining data consistency when performing related updates. Learn more about transaction in [etcd's Learning](https://etcd.io/docs/v3.6/learning/api/#transaction).
```

Comment:
Two fixes: This should be the 3.5 version, and there's a typo in the URL, and the link should have a different name:

```suggestion
Transactions in etcd allow you to execute multiple operations atomically, ensuring that either all operations are applied or none are. This is crucial for maintaining data consistency when performing related updates. Learn more about transactions in [the API documentation](https://etcd.io/docs/v3.5/learning/api#transaction).
```